plugins/janus_audiobridge.c

```diff
@@ -1256,6 +1279,21 @@ void janus_audiobridge_destroy_session(janus_plugin_session *handle, int *error)
 	return;
 }
 
+static void janus_audiobridge_notify_participants(janus_audiobridge_participant *participant, json_t *msg) {
+	/* participant->room->participants_mutex has to be locked. */
```

Comment:
The AudioBridge plugin doesn't have any `participants_mutex`. I guess this should refer to the `mutex` property instead, which should be locked when you call `janus_audiobridge_notify_participants` below.